### PR TITLE
NF: First pass at a dipy DTI interface.

### PR DESCRIPTION
**Pull request by arokem**
Hi, following up on conversation with @chrisfilo earlier today, here is a first pass at trying to extend dipy functionality in nipype. Am I doing this right?

### CODE REVIEW

**Comment by chrisfilo** [comment]
Very decent start! Please add "DTI" to __init__ of this module to fix some tests.

File: nipype/interfaces/dipy/tensors.py [hunk 1]
```diff
@@ -63,38 +164,18 @@ class TensorMode(BaseInterface):
     >>> mode.inputs.in_file = 'diffusion.nii'
     >>> mode.inputs.bvecs = 'bvecs'
     >>> mode.inputs.bvals = 'bvals'
+    >>> mode.inputs.mask = 'wm_mask.nii'
```
Comment (chrisfilo) [inline nipype/interfaces/dipy/tensors.py]: it should be "mask_file"

**Comment by chrisfilo** [comment]
Also update autotests for tensormode

**Comment by arokem** [comment]
Thanks for taking a look. How do I generate these autotests?

**Comment by chrisfilo** [comment]
make specs

**Comment by arokem** [comment]
Tests seem to be passing now. How's this?

**Pull request merged**